koku/api/report/all/openshift/provider_map.py

```diff
@@ -395,7 +395,7 @@ def __init__(self, provider, report_type):
                             "cost_usage": Sum(Value(0, output_field=DecimalField())),
                             "cost_markup": Sum(Coalesce(F("markup_cost"), Value(0, output_field=DecimalField()))),
                             "cost_units": Coalesce(Max("currency_code"), Value("USD")),
-                            "count": Count("resource_id", distinct=True),
+                            "count": ArrayAgg("resource_id", distinct=True),
```

Comment:
ArrayAgg here will return a list of resource ids. Is that what we want?

Reply:
Hey Andrew! 

This change stemmed from some smoke test failures where the count was unusually high. I discovered that the count of the resource_ids was essentially loosing the "distinct" during the pandas implementation. Therefore in order to deduplicate the results again, I changed it to a list and [implemented some deduplication](https://github.com/project-koku/koku/pull/3708/commits/df51675ea0fda099a3bd3d1899e02dff88336d50#diff-7856c4ba653f802f9901aa185f9dfcd3754d01fe20a6fa824b6443d31b88d48aR715) for when we are calculating the totals. 

I am open to some groovy pandas alternatives if they exist, I just couldn't figure them out myself 😂

Reply:
@myersCody I think it's likely because the count term is being summed, when we want the maximum value for count. 

`aggs = {col: ["max"] if "units" in col else ["sum"] for col in annotations if col not in skip_columns}`